### PR TITLE
OG-Core demo day

### PR DESCRIPTION
**Pull request by jdebacker**
Blog post to follow the OG-Core demo day presentation.

### CODE REVIEW

**Comment by jdebacker** [comment]
Screenshot:

Screen Shot 2021-11-02 at 3 20 52 PM

**Comment by jdebacker** [comment]
@rickecon would you mind giving this post a quick review?  Thanks!

**Comment by rickecon** [comment]
@jdebacker. I am working on this right now.

File: _posts/2021-11-01-demo-day-21-og_core.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- macroeconomics
+- overlapping-generations
+layout: post
+toc: true
+---
+
+{% include youtube.html content='https://youtu.be/y1vSgmEtgxs' %}
+---
+
+The [OG-Core](https://pslmodels.github.io/OG-Core/content/intro/intro.html) model is a general equilibrium, overlapping generations (OG) model suitable for evaluating fiscal policy.
```
Comment (rickecon) [inline _posts/2021-11-01-demo-day-21-og_core.md:17]: @jdebacker. I would change this URL to just `https://pslmodels.github.io/OG-Core/`.

File: _posts/2021-11-01-demo-day-21-og_core.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Since the work of Alan Auerbach and Larry Kotlikoff in the 1980s, this class of model has become a standard in the macroeconomic analysis of tax and spending policy.
```
Comment (rickecon) [inline _posts/2021-11-01-demo-day-21-og_core.md:18]: @jdebacker. I would put "Laurence Kotlikoff" here, although "Larry" is more familiar.

File: _posts/2021-11-01-demo-day-21-og_core.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+This is for good reason.
+OG models are able to capture the impacts of taxes and spending in the short and long run, examine incidence of policy across generations of people (not just short run or steady state analysis of a cross-section of the economy), and capture important economic dynamics (e.g., crowding out effects of deficit-financed policy).
+
+In the PSL Demo Day presentation linked to above, I cover the basics of OG-Core: its history, its API, and how country-specific models can use OG-Core as a dependency.
```
Comment (rickecon) [inline _posts/2021-11-01-demo-day-21-og_core.md:22]: @jdebacker . Just "linked above".

**Comment by rickecon** [comment]
@jdebacker. I just had three small comments, each of which you could disregard and the post would be fine. LGTM.

**Comment by jdebacker** [comment]
Thanks @rickecon!

**Pull request merged**